wpcom-legacy-redirector.php

```diff
@@ -135,9 +154,9 @@ static function get_redirect_uri( $url ) {
 
 				return false;
 			} elseif ( 0 !== $redirect_post->post_parent ) {
-				return get_permalink( $redirect_post->post_parent );
+				return add_query_arg( $protected_param_values, get_permalink( $redirect_post->post_parent ) ); // Add Whitelisted Params to the Redirect URL.
```

Comment:
Is `$protected_param_values` going to be defined here? I'm not following how the value gets passed from `get_redirect_uri()` to here.